src/core/symbology-ng/qgsconcentriclegendsymbol.h

```diff
@@ -33,6 +33,7 @@ typedef QPair< QString, double > LabelValue;
  * \class QgsConcentricLegendSymbol
  * \brief Symbol used to display the diagram size legend.
  * \note Added in version 3.0
+ * \since QGIS 3.0
```

Comment:
The `\note Added in version 3.0` needs to be removed (it's replaced by `\since`)